pdr_backend/ppss/base_ss.py

```diff
@@ -174,3 +183,36 @@ def get_feed_from_candidates(
                 return feed
 
         return None
+
```

Comment:
MockExchange stuff should be in its own module, eg mock_exchange.py, not base_ss.py.